src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,29 @@
+(function(transport, global, console) {
+    "use strict";
+
+    function _log(s){
+        transport.send("bramble-console", s);
+    }
+
+    // Bind _log to iframe console.
+    console.log = _log;
+
+    // Implement other Log Levels to console.log
+    console.debug = console.log;
+    console.info = console.log;
+    console.warn = console.log;
+
+    if (!console.assert) {
+        console.assert = function() {
+            var args = Array.from(arguments).slice();
+            var expr = args.shift();
+            if (!expr) {
+                args[0] = "Assertion Failed: " + args[0];
+                console.error.apply(console,args);
+            }
+        };
+    }
+
```

Comment:
I still think you should go look at all the other `console.*` methods and add them, see https://github.com/firebug/firebug-lite/blob/master/content/firebug1.4/consoleInjector.js#L260 and https://developer.mozilla.org/en/docs/Web/API/console